config/containers/container-networking.md

```diff
@@ -56,8 +56,7 @@ flag to specify an additional network alias for the container on that network.
 
 ## DNS services
 
-By default, a container inherits the DNS settings of the Docker daemon,
-including the `/etc/hosts` and `/etc/resolv.conf`.You can override these
+By default, a container inherits the DNS settings of the Docker daemon by copying the config file `/etc/resolv.conf` to the container. The file `/etc/hosts` however, are not inherited. To pass additional hosts into your container, see [run command](../../engine/reference/commandline/run.md#add-entries-to-container-hosts-file---add-host). You can override these
```

Comment:
Thanks! Wondering if we should also mention the embedded DNS here. Looks like more details were original in this section (https://github.com/docker/docker.github.io/blob/v17.09/engine/userguide/networking/configure-dns.md), but got removed as part of a rewrite in https://github.com/docker/docker.github.io/pull/5936)


Something like;

```suggestion
By default, a container inherits the DNS settings of the host, as defined in the
`/etc/resolv.conf` configuration file. Containers that use the default `bridge`
network get a copy of this file, whereas containers that use a
[custom network](../../network/network-tutorial-standalone.md#use-user-defined-bridge-networks)
use Docker's embedded DNS server, which forwards external DNS lookups to the DNS
servers configured on the host.

Custom hosts defined in `/etc/hosts` are not inherited. To pass additional hosts
into your container, refer to [add entries to container hosts file](../../engine/reference/commandline/run.md#add-entries-to-container-hosts-file---add-host)
in the `docker run` reference documentation. You can override these settings on
a per-container basis.
```


@usha-mandya

Reply:
+1 for mentioning the embedded DNS. Stumbled across the embedded DNS while debugging my issue with custom hosts and it helps a lot in explaining the behavior I observed. Thank you for the correction and suggestion!